### ISSUE TITLE
emotion@10 throws styled.div is not a function

### ISSUE THREAD

**Issue by seawatts** [emotion-js/emotion#1056]
- `emotion` version: 10.0.0
- `react` version: 16.6.3

I have been using `CreateStyled` type imported from `react-emotion` to create my typed styled theme per the typescript docs. https://emotion.sh/docs/typescript#define-a-theme. However, after upgrading to `emotion@10` I am now getting the following error.

Relevant code.

```ts
// styles/theme.ts
import styled, { CreateStyled } from '@emotion/styled'

export const theme = {
  black: 'black',
  maxWidth: '100%'
};

export default styled as CreateStyled<typeof theme>;
```

```js
// Error 
TypeError: _styles_theme__WEBPACK_IMPORTED_MODULE_2__.default.div is not a function
```

**Comment by smdern** [comment]
just FYI I was able to get it to work by doing the following

```
import * as styled from '@emotion/styled'
import { Theme } from './theme'

export default styled.default as styled.CreateStyled<Theme>
```

**Comment by Andarist** [comment]
This seems like some kind of broken setup. This (`import * as styled from '@emotion/styled'`) is not correct - `@emotion/styled` has ONLY default export.

**Comment by smdern** [comment]
Agreed. Something isn't right. The other solution I came up with was aliasing `@emotion/styled` to `'@emotion/styled/dist/styled.esm.js'`

But this problem seems to be not isolated.

**Comment by mitchellhamilton** [comment]
Do you use the babel plugin?

**Comment by jennyvallon** [comment]
For archival sake, I am having the same issue years later. In days of scouring the internet to find a solution https://github.com/emotion-js/emotion/issues/1056#issuecomment-444886272 was the only bit of information that worked.

**Comment by barathrajd** [comment]
@makaratzisiordanis, Any Solution for emotion to be shared with ModuleFederationPlugin?
